src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -68,9 +69,19 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     WorkspaceStage internalStage = WorkspaceStage.fromApiModel(requestStage);
     Optional<SpendProfileId> spendProfileId =
         Optional.ofNullable(body.getSpendProfile()).map(SpendProfileId::create);
+    // If clients do not provide an operation ID, we generate one instead.
+    String operationId =
+        body.getOperationId() != null ? body.getOperationId() : UUID.randomUUID().toString();
+
+    WorkspaceRequest internalRequest =
+        WorkspaceRequest.builder()
+            .workspaceId(body.getId())
+            .operationId(operationId)
```

Comment:
nit: We've been inconsistent so far (sorry), but we should probably pick a single term for 'job' vs 'operation'. No time like the present, so let's do 'jobId' here to correspond with the job service.

Reply:
we're already using `jobId` in JobService, so I think that's the right standard